### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by cameronbrill**

### CODE REVIEW

**Review by OneWebDev — APPROVED** [review]

**Review by eightants — CHANGES_REQUESTED** [review]
None of the changes on the landing page was made
image

**Comment by eightants** [comment]
image
Also, space between paragraphs are really huge

File: th/index.html [hunk 1]
```diff
@@ -97,7 +97,7 @@
             </div>
             <div>
                 <div class="page-content">
-                    <div class="page-content-panel page-content-img">
+                    <div class="page-content-panel page-content-img lg:w-1/2">
```
Comment (eightants) [inline th/index.html:100]: The only line needed for #204

**Review by eightants — DISMISSED** [review]

File: th/index.html (hunk 1 above)
Comment (cameronbrill) [reply th/index.html:100]: :o

**Review by eightants — APPROVED** [review]

**Pull request merged**